tests/test_ops_quota.py

```diff
@@ -16,6 +16,7 @@ def issue_request(client):
 
 
 # Tests
+@pytest.mark.skip(reason="FIXME: Currently fails")
 def test_mock_quota_exceeded(all_clients, monkeypatch):
     monkeypatch.setattr(all_clients, "__service_url_prefix__", APIARY_URL)
     errors = {
```

Comment:
The reason why this test fails is:

```python
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://opsv31.docs.apiary.io//individual-per-hour-exceeded/publication/docdb/biblio
```

Can you do anything about it, or tell me how I would approach that issue, @gsong?